src/main.rs

```diff
@@ -152,7 +152,7 @@ struct Readouts {
     kernel_readout: KernelReadout,
 }
 
-fn get_info(info: &PfetchInfo, readouts: &Readouts) -> Option<String> {
+fn get_info(info: &PfetchInfo, readouts: &Readouts, skip_slow: bool) -> Option<String> {
```

Comment:
Rename `skip_slow` to `skip_slow_package_managers` here to make clear what is skipped